tests/react-integration-test/check-japgolly/c/componentstest/build.sbt

```diff
@@ -1,12 +1,12 @@
 organization := "org.scalablytyped"
 name := "componentstest"
-version := "0.0-unknown-3f950c"
+version := "0.0-unknown-8e3fb7"
 scalaVersion := "2.13.3"
 enablePlugins(ScalaJSPlugin)
 libraryDependencies ++= Seq(
   "com.github.japgolly.scalajs-react" %%% "core" % "1.7.5",
   "com.olvind" %%% "scalablytyped-runtime" % "2.4.0",
-  "org.scalablytyped" %%% "react" % "16.9.2-42070f",
+  "org.scalablytyped" %%% "react" % "16.9.2-803716",
```

Comment:
I think you can make this project to depend on the main one, so that you don't need to hardcode the version

Reply:
Everything underneath `tests` is generated code, which works a snapshot tests to track changes in output over time :)